kOS/pegas_util.ks

```diff
@@ -415,6 +415,14 @@ FUNCTION setUserEvents {
 	userEventHandler().
 }.
 
+//	Setup a trigger listening for mesages from other CPUs
+FUNCTION setComms {
+	WHEN NOT CORE:MESSAGES:EMPTY THEN {
+		SET commsEventFlag TO TRUE.
+		RETURN TRUE.				//	Preserves the trigger
```

Comment:
Will need to change this part, if message isn't handled straight away (e.g. if in the middle of setting up UPFG when messages are not being checked), this trigger will fire up every tick.
Instead of preserving the trigger, I could maybe call `setComms()` again after handling and removing a message from queue.

Reply:
Done.